src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/DatabasesOperationsExtensions.cs

```diff
@@ -0,0 +1,1149 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
- Can you provide a link to your PR in rest-api-specs?

-- project.json.  
    - You need to insert the proper metadata for a package, including Description, Summary, release notes
   - You need to have a new version, it must be a preview version.
   - Consider using a different package name and different namespaces - this will make it easier for customers to transition, and easier for you guys to transition your PowerShell cmdlets.


- since you are making changes, you should refer to your swagger document through the last commit, rather than a floasting reference to the branch.  like this:  https://github.com/Azure/azure-rest-api-specs/blob/3b5b4e6d836ebdf8082fe8ca78a23699aaaf7340/arm-sql/compositeSql.json